addons/sairam.quadtree/QuadTree.gd

```diff
@@ -195,14 +216,13 @@ func _get_child(body_bounds: AABB) -> QuadTree:
 	
 	Gets the child that incorporates itself in the body_bounds passed.
 	"""
-	if body_bounds.end.z < _bounds.end.z:
-		if body_bounds.end.x < _bounds.end.x:
+	if body_bounds.end.z < _bounds.position.z + _center.z:
```

Comment:
Shouldn't it be `end`?

Reply:
You want to go up/down and left/right based on splitting the bounds in half. If it's less than half, go down/up, etc.